underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -463,6 +480,29 @@ public long getSpace(String path, SpaceType type) throws IOException {
     return space;
   }
 
+  @Override
+  public UfsStatus getStatus(String path, GetFileStatusOptions options) throws IOException {
```

Comment:
Does `S3AUnderFileSystem` provide something similar to the HDFS Content Hash?

Reply:
yes, we would add that for s3 & gcs